src/pages/Products.js

```diff
@@ -178,18 +199,35 @@ const Products = () => {
 							</h2>
 						</div>
 					)}
-					<div className="grid md:grid-cols-2 xl:grid-cols-4 gap-4 w-full p-0 justify-evenly xl:justify-around">
-						{currentProducts?.length ? (
-							currentProducts.map((product) => (
-								<ProductItem data={product} key={product.id} />
-							))
-						) : (
-							<h1>No hay productos usando estos argumentos</h1>
-						)}
-					</div>
+					{
+						currentProducts.length === 0
+						? <h1 className="text-center">No hay productos usando estos argumentos</h1>
+						:
+						<>
+							<div className="grid md:grid-cols-2 xl:grid-cols-4 gap-4 w-full p-0 justify-evenly xl:justify-around">
+								{
+									currentPosts.map((product) => (
+										<ProductItem data={product} key={product.id} />
```

Comment:
When adding the key is a good practice to make it more unique and more descriptive by a descriptive suffix, for example:
```suggestion
										<ProductItem data={product} key={`product-item-${product.id}`} />
```